src/main/java/duke/command/CommandParser.java

```diff
@@ -0,0 +1,23 @@
+package duke.command;
+
+import duke.TaskList;
+import duke.models.Task;
+
+public class CommandParser {
+
+    public static String parseCommand(CommandResult commandResult, TaskList taskList, Task t) {
+        switch (commandResult) {
+            case SUCCESSFUL_ADD:
+                return "Got it. I've added this task:\n" + t + "\nNow you have";
+            case SUCCESSFUL_DELETE:
+                break;
+            case SUCCESSFUL_MARK:
+                return "Nice! I've marked this task as done\n" + t;
+            case SUCCESSFUL_UNMARK:
+                break;
```

Comment:
Should the switch case statements follow a different format? I think you can configure the settings of your ide to suit the coding standards for this module. 😃 
```suggestion
        switch (commandResult) {
        case SUCCESSFUL_ADD:
            return "Got it. I've added this task:\n" + t + "\nNow you have";
        case SUCCESSFUL_DELETE:
            break;
        case SUCCESSFUL_MARK:
            return "Nice! I've marked this task as done\n" + t;
        case SUCCESSFUL_UNMARK:
            break;
```